anthos-bm-gcp-terraform/docs/nfs.md

```diff
@@ -1,87 +1,64 @@
 ## NFS Shared Storage
 
-An optional NFS server can be provisioned to provide shared storage to the **Anthos on bare metal** cluster.  In the standard [Quick Starter](quickstart.md) method, commands are provided to install the NFS Container Storage Interface (CSI) driver, in the [All in one install](one_click_install.md) or [Manual LB install](manuallb_install.md) methods this is done automatically.
+An optional NFS server can be provisioned to provide shared storage to the **Anthos on bare metal** cluster. If you follow the standard [Quick Starter](quickstart.md) guide, then the commands required to install the **NFS Container Storage Interface (CSI)** driver are provided as part of the Terraform output. If you followed one of [All in one install](one_click_install.md) or [Manual LB install](manuallb_install.md) guides then everything is setup automatically.
```

Comment:
are there any links we can provide in cloud docs for what NFS is to the user?

Reply:
Good point. Maybe link to this doc: https://kubernetes.io/docs/concepts/storage/volumes/#nfs. It also points to an example.

Reply:
Added link to the OSS K8s doc in https://github.com/GoogleCloudPlatform/anthos-samples/pull/318/commits/f016e20455ef97a55bfd109c8b1799dbbbf9a13b